lib/utils.js

```diff
@@ -238,4 +378,6 @@ module.exports = {
   time,
   config,
   format,
+  fill,
```

Comment:
fill and references uses routes/utils and also `singleItem`. And we cannot create dependency of utils in content/utils.